.travis.yml

```diff
@@ -0,0 +1,57 @@
+# Travis-CI for Iknaia
+
+language: cpp
+
+sudo: required
+
+# Compiler selection
+compiler:
+  - gcc
+
+# Before Installing
+before_install:
+  - if [ "$CXX" == "g++" ]; then sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test && sudo apt-get update; fi
+
+# Install dependencies
+install:
+  # Installing gcc-4.7
+  - if [ "$CXX" = "g++" ]; then sudo apt-get -y install g++-4.7; fi
```

Comment:
Is g++-4.7 really required? Plus some other packages below. We should try to cope with whatever default systems installs. If there is a requirement for a specific package, install locally or in a confined environment so not to disturb other applications.

Initially, this can be just in ubuntu. In the future, we can extend to other flavors of linux (fedora, archlinux, etc.)